zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/web/rest/ZaakObjectResource.kt

```diff
@@ -54,7 +54,11 @@ class ZaakObjectResource(
                 it.url,
                 it.record.index,
                 it.record.registrationAt,
-                it.record.data?.get("title") as String?
+                if (it.record.data?.get("title")?.isTextual == true) {
```

Comment:
Feels like this stuff belongs either in a mapper or in the object itself, and not the resource.